pkg/providers/ovirt/templates/template-handler.go

```diff
@@ -25,8 +25,8 @@ const (
 	templateNamespaceLabel = "vm.kubevirt.io/template-namespace"
 )
 
-// TemplateFinder attempts to find a template based on given parameters
-type TemplateFinder struct {
+// TemplateHandler attempts to find a template based on given parameters
```

Comment:
pls add a notation about the processing part of it

Reply:
ack